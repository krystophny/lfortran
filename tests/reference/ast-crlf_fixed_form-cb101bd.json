{
    "basename": "ast-crlf_fixed_form-cb101bd",
    "cmd": "lfortran --fixed-form --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/crlf_fixed_form.f",
    "infile_hash": "e96bb5c995d0b2ee78605b345ce4528554efe2c7d340ee2ca6474648",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-crlf_fixed_form-cb101bd.stdout",
    "stdout_hash": "f3e0c26df2bacd87f94a6d49912b9d7fdbaa7b937a9e99475d01ed0d",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}