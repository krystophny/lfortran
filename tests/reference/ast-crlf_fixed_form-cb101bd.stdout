(TranslationUnit
    [(Program
        test
        ()
        []
        []
        []
        [(Continue
            10
            ()
        )
        (Print
            0
            ()
            [(String "hello\r world")]
            ()
        )]
        []
    )]
)
